tests/data/idls/ThriftTest2.thrift

```diff
@@ -0,0 +1,30 @@
+union Value {
+    1: string stringValue
+    2: i32 integerValue
+}
+
+struct Item {
+    1: required string key
+    2: required Value value
+}
+
+exception ItemAlreadyExists {
+    1: required Item item
+    2: optional string message
+}
+
+exception ItemDoesNotExist {
+    1: required string key
+}
+
+service Service {
+    // oneway void putItemAsync(1: Item item);
+    // TODO: oneway not yet supported
+
+    void putItem(1: Item item, 2: bool failIfPresent)
+         throws (1: ItemAlreadyExists alreadyExists);
+    Item getItem(1: string key)
+         throws (1: ItemDoesNotExist doesNotExist);
+
+    bool healthy();
+}
```

Comment:
this thrift file is now being used in place of a pytest fixture that wrote a thrift file to disk then called the thrift cli to generate code